src/open_inwoner/components/templates/components/UserFeed/UserFeed.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n icon_tags %}
+
+{# if userfeed data is true then show component #}
+<section class="userfeed">
+    <h2 class="h2 {# if userfeed true #}indicator{# endif #}">
+        {% trans "Aanvragen updates" %}
+    </h2>
+    <div class="userfeed__summery">
+        <ul class="userfeed__list">
+            <li class="userfeed__list-item">
+                <p class="userfeed__list-paragraph">
+                    Bij <span>X</span> aanvraag{# if statuses > 1 #}aanvragen{# endif #} is uw actie nodig
+                </p>
+            </li>
+            <li class="userfeed__list-item">
+                <p class="userfeed__list-paragraph">
+                    Bij <span>X</span> aanvraag{# if statuses > 1 #}aanvragen{# endif #} is de status gewijzigd
+                </p>
+            </li>
+        </ul>
+    </div>
+    <div class="card-container card-container--columns-2">
+        <a href="{{ userfeedbackend }}" class="card {# if action-needed true #}card--status{# endif #}">
+            <div class="card__body card__body--tabled">
+                <p class="p tabled__value">Bijzondere bijstand aanvragen</p>
```

Comment:
Are we still translating?

Reply:
This text will come from the back-end later on and will be completely different, depending on a case with a certain action-text, so right now it is all still static (this template is not used anywhere now, so no need for translation right here).